test/travis-install.bash

```diff
@@ -27,10 +34,9 @@ if [[ $CH_BUILDER = buildah* ]]; then
     mkdir /usr/local/src/go
     pushd /usr/local/src/go
     export GOPATH=$PWD
-    git clone https://github.com/containers/buildah \
-              src/github.com/containers/buildah
+    git clone $buildah_repo src/github.com/containers/buildah
```

Comment:
Clone and checkout can be done in a single line with "--branch", but the current way should work fine as well.
```suggestion
    git clone $buildah_repo src/github.com/containers/buildah --branch $buildah_branch
```

Reply:
Yeah, you're right. One gotcha is that you can't give a commit hash to `--branch`, for reasons that seem silly to me. Because it works already and for that reason, I'm going to leave it as is, but good catch.